src/shared/components/Dropdown/Dropdown.Item.tsx

```diff
@@ -0,0 +1,60 @@
+'use client';
+
+import { ReactNode } from 'react';
+
+import { cn } from '@/shared/libs/cn';
+
+import { useDropdownContext } from './DropdownContext';
+
+/**
+ * DropdownItem
+ *
+ * Dropdown 내 개별 항목 컴포넌트입니다.
+ *
+ * 클릭 시 `onClick` 핸들러가 실행되고, 드롭다운 메뉴가 닫힙니다.
+ *
+ * @param {ReactNode | string} props.children - 버튼 안에 들어갈 콘텐츠
+ * @param {() => void} props.onClick - 버튼 클릭 시 실행될 함수
+ * @param {string} [props.itemClassName] - 버튼에 추가로 적용할 클래스명 (선택사항)
+ *
+ * @example
+ * ```tsx
+ * <DropdownItem onClick={() => console.log('클릭됨!')}>
+ *   메뉴 1
+ * </DropdownItem>
+ * ```
+ *
+ */
+export default function DropdownItem({
+  children,
+  onClick,
+  itemClassName,
+}: {
+  children: string | ReactNode;
+  onClick: () => void;
+  itemClassName?: string;
+}) {
+  const { setIsOpen } = useDropdownContext();
+
+  const handleClick = (e: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
위에 `children`은 `ReactNode`를 import해서 사용해주셨는데, 이 부분도 같은 맥락으로  `MouseEvent`도 import 해서 쓰시면 일관성 있는 코드를 작성할 수 있을 거 같습니다!

```suggestion
  const handleClick = (e: MouseEvent<HTMLButtonElement>) => {
```

Reply:
오 꼼꼼하게 살펴봐주셔서 감사합니다 :)
확실히 일관성있게 작성해두는게 좋을 것 같아 보이네요!
해당 사항 참고해서 수정해두도록 하겠습니다! 👍